project/Dependencies.scala

```diff
@@ -139,14 +139,18 @@ object Dependencies {
       )
   }
 
-  class MunitCross(munitVersion: String) {
+  class MunitCross(val munitVersion: String) {
     val core: Def.Initialize[ModuleID] =
       Def.setting("org.scalameta" %%% "munit" % munitVersion)
     val scalacheck: Def.Initialize[ModuleID] =
       Def.setting("org.scalameta" %%% "munit-scalacheck" % munitVersion)
   }
   object Munit extends MunitCross("0.7.29")
   object MunitMilestone extends MunitCross("1.0.0-M6")
+  object MunitV1 extends MunitCross("1.0.0") {
```

Comment:
may as well use 1.0.0 everywhere.

Reply:
Tried that, but 1.0.0 is not published for 0.4.x. This would involve upgrading native to 0.5.x and I'm not sure about cats-effect support there.